### ISSUE TITLE
roxygen2 documentation printing outside of page margins

### ISSUE THREAD

**Issue by mccroweyclinton-EPA** [r-lib/roxygen2#1206]
Good morning I have created a new package for R, for some reason the @family tags is causing function names in the See Also sections of some functions to display outside of the page margins. Since I am not sure how or even if github supports adding an attachment to an issue. I will have to directly link to my github project in order to give and example.

https://github.com/USEPA/RAQSAPI/issues/2.

If you compile the documentation that is included with this package you can see that the function names extend past the margins of the page.

**Comment by gaborcsardi** [comment]
Can you please show us a example that we can reproduce?

**Comment by mccroweyclinton-EPA** [comment]
[RAQSAPI manual sample.pdf](https://github.com/r-lib/roxygen2/files/6037521/RAQSAPI.manual.sample.pdf)

OK, I figured out how to add an attachment to an issue. As you can see in the attached file function names are being clipped by the page margins.

**Comment by gaborcsardi** [comment]
This is the documentation roxygen2 generates:

```
\seealso{
Other Aggregate _by_county functions: 
\code{\link{aqs_dailysummary_by_county}()},
\code{\link{aqs_monitors_by_county}()},
\code{\link{aqs_qa_blanks_by_county}()},
\code{\link{aqs_qa_collocated_assessments_by_county}()},
\code{\link{aqs_qa_flowrateaudit_by_county}()},
\code{\link{aqs_qa_flowrateverification_by_county}()},
\code{\link{aqs_qa_one_point_qc_by_county}()},
\code{\link{aqs_qa_pep_audit_by_county}()},
\code{\link{aqs_sampledata_by_county}()},
\code{\link{aqs_transactionsample_by_county}()}
}
``` https://github.com/USEPA/RAQSAPI/blob/166c1f30d44a6c7512d4438d07102ef6c80a2a59/man/aqs_annualsummary_by_county.Rd#L93-L104

R and LateX creates the PDF, and seemingly the default is to create justified text, so you'll get what you get. In theory you could make it ragged right with an extra LaTEX package, but AFAIR R does not allow loading extra LaTeX packages when formatting the manual.

In summary, I don't think there is a way to fix this in roxygen2, but I'll be happy to be proven wrong.

**Comment by mccroweyclinton-EPA** [comment]
Thank you for at least looking into this. If something comes up in the future that I can do to fix this please let me know.
